ui/lib/pki/addon/components/page/pki-configuration-details.hbs

```diff
@@ -1,25 +1,56 @@
-<h2 class="title has-bottom-margin-xs has-top-margin-m is-4 has-border-bottom-light has-bottom-padding-s">
-  Global URLs
-</h2>
-<InfoTableRow @label="Issuing certificates" @value={{or @urls.issuingCertificates "None"}} />
-<InfoTableRow
-  @label="CRL distribution points"
-  @value={{if @urls.crlDistributionPoints @urls.crlDistributionPoints "None"}}
-/>
+<Toolbar>
+  <ToolbarActions>
+    {{#if @canDeleteAllIssuers}}
+      <button
+        type="button"
+        class="toolbar-link"
+        {{on "click" (fn (mut this.showDeleteAllIssuers) true)}}
+        data-test-delete-all-issuers-link
+      >
+        Delete all issuers
+        <Icon @name="chevron-right" />
 
-<h2 class="title has-bottom-margin-xs has-top-margin-xl is-4 has-border-bottom-light has-bottom-padding-s">
-  Certificate Revocation List (CRL)
-</h2>
-<InfoTableRow @label="Expiry" @value={{@crl.expiry}} />
-<InfoTableRow @label="Auto-rebuild" @value={{if @crl.autoRebuild "On" "Off"}} />
+      </button>
+      <div class="toolbar-separator"></div>
+    {{/if}}
+    <ToolbarLink @route="configuration.tidy">
+      Tidy
+    </ToolbarLink>
+    <ToolbarLink @route="configuration.edit">
+      Edit configuration
+    </ToolbarLink>
+  </ToolbarActions>
+</Toolbar>
 
-<h2 class="title has-bottom-margin-xs has-top-margin-xl is-4 has-border-bottom-light has-bottom-padding-s">
-  Online Certificate Status Protocol (OCSP)
-</h2>
-<InfoTableRow @label="Responder APIs" @value={{if @crl.ocspDisable "Disabled" "Enabled"}} />
-<InfoTableRow @label="Interval" @value={{@crl.ocspExpiry}} />
+{{#if (not (eq @urls 403))}}
+  <h2 class="title has-bottom-margin-xs has-top-margin-m is-4 has-border-bottom-light has-bottom-padding-s">
```

Comment:
nit, could you move the is-4 next to title? The is-4 helper affects titles only. e.g. title.is-4.